src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapListMessagesBodyStructureScenario.scala

```diff
@@ -27,7 +27,7 @@ class ImapListMessagesBodyStructureScenario {
             .select("${mailboxes.random()}")
             .check(ok))
           .exec(imap("fetch")
-            .fetch(MessageRanges(Range(1, mailsFetched)), AttributeList("BODYSTRUCTURE"))
+            .fetch(MessageRanges(From(1L)), AttributeList("BODYSTRUCTURE"))
```

Comment:
I'm reluctant for that change, as the performance depends highly on the count of fetched items...

Maybe if doable we could provision more messages?

Maybe we could decrease the amount of mail fetched?

Reply:
I reverted the code and put a limit of 50 for mails fetched. As we saw on the provisioning by checking Cassandra, we have an average of a 100 messages per mailbox more or less 10 on it. With a limit of 50 we are pretty sure it's not gonna fail for this. 50 should be enough as well for now.